exporter/containerimage/writer.go

```diff
@@ -150,6 +151,8 @@ func (ic *ImageWriter) Commit(ctx context.Context, inp exporter.Source, oci bool
 }
 
 func (ic *ImageWriter) exportLayers(ctx context.Context, compressionType compression.Type, forceCompression bool, s session.Group, refs ...cache.ImmutableRef) ([]solver.Remote, error) {
+	span, ctx := tracing.StartSpan(ctx, "export layers")
```

Comment:
we might want to capture some of the argument here, `compressionType` seems interesting, maybe something like:
```
span, ctx := tracing.StartSpan(ctx, "export layers", trace.WithAttributes(
    attribute.String("exportLayers.compressionType", compressionType.String()),
    attribute.Bool("exportLayers.forceCompression", bool),
))
```